_data/health.yml

```
@@ -2,10 +2,9 @@ websites:
     - name: 23andMe
       url: https://www.23andme.com/
       img: 23andme.png
-      tfa: No
-      twitter: 23andMesupport
-      facebook: 23andMe
-      email_address: customercare@23andme.com
+      tfa: Yes
+      software: Yes
+      doc: https://customercare.23andme.com/hc/en-us/articles/360034119874-Adding-2-Step-Verification-to-Your-23andMe-Account
```

```suggestion
      doc: https://customercare.23andme.com/hc/en-us/articles/360034119874
```
HelpCenter article do not require a slug at all.

---

@markdrayton Would you please commit my suggestions as I don't have write permissions to your fork of our repository. Thanks!
// Kai

---

Done! Thanks for the review.